app/src/main/cpp/skyline/nce/guest.cpp

```diff
@@ -220,7 +220,7 @@ namespace skyline::guest {
         volatile ThreadContext *ctx;
         asm("MRS %0, TPIDR_EL0":"=r"(ctx));
 
-        for (u8 index = 0; index < 30; index++)
+        for (u8 index{}; index < 30; index++)
```

Comment:
maybe make the reg count a constant? or use regs.size()

Reply:
It's already one in NCE and `guest.cpp` is going to be removed so I'll hold out on adding